src/manager/manager.c

```diff
@@ -736,13 +744,56 @@ static int manager_method_create_monitor(sd_bus_message *m, void *userdata, UNUS
         return 1;
 }
 
+/*************************************************************************
+ ************** org.containers.hirte.Manager.EnableMetrics ***************
+ ************************************************************************/
+static int manager_method_metrics_enable(sd_bus_message *m, void *userdata, UNUSED sd_bus_error *ret_error) {
+        Manager *manager = userdata;
+        Node *node = NULL;
+        if (manager->metrics_enabled) {
+                hirte_log_debug("Metrics already enabled");
+                return sd_bus_reply_method_return(m, "s", "Metrics already enabled");
+        }
+        if (!metrics_api_service_register(manager)) {
+                return sd_bus_reply_method_errorf(m, SD_BUS_ERROR_FAILED, "Internal error");
+        }
+        manager->metrics_enabled = true;
+        LIST_FOREACH(nodes, node, manager->nodes) {
+                node_enable_metrics(node);
+        }
+        hirte_log_debug("Metrics enabled");
+        return sd_bus_reply_method_return(m, "s", "Done");
+}
+
+/*************************************************************************
+ ************** org.containers.hirte.Manager.DisableMetrics **************
+ ************************************************************************/
+static int manager_method_metrics_disable(sd_bus_message *m, void *userdata, UNUSED sd_bus_error *ret_error) {
+        Manager *manager = userdata;
+        Node *node = NULL;
+        if (!manager->metrics_enabled) {
+                hirte_log_debug("Metrics already disabled");
+                return sd_bus_reply_method_return(m, "s", "Metrics already disabled");
+        }
+        sd_bus_slot_unrefp(&manager->metrics_slot);
+        manager->metrics_slot = NULL;
+        manager->metrics_enabled = false;
+        LIST_FOREACH(nodes, node, manager->nodes) {
+                node_disable_metrics(node);
+        }
+        hirte_log_debug("Metrics disabled");
+        return sd_bus_reply_method_return(m, "s", "Done");
+}
+
 static const sd_bus_vtable manager_vtable[] = {
         SD_BUS_VTABLE_START(0),
         SD_BUS_METHOD("Ping", "s", "s", manager_method_ping, 0),
         SD_BUS_METHOD("ListUnits", "", NODE_AND_UNIT_INFO_STRUCT_ARRAY_TYPESTRING, manager_method_list_units, 0),
         SD_BUS_METHOD("ListNodes", "", "a(sos)", manager_method_list_nodes, 0),
         SD_BUS_METHOD("GetNode", "s", "o", manager_method_get_node, 0),
         SD_BUS_METHOD("CreateMonitor", "", "o", manager_method_create_monitor, 0),
+        SD_BUS_METHOD("EnableMetrics", "", "s", manager_method_metrics_enable, 0),
+        SD_BUS_METHOD("DisableMetrics", "", "s", manager_method_metrics_disable, 0),
```

Comment:
```suggestion
        SD_BUS_METHOD("EnableMetrics", "", "", manager_method_metrics_enable, 0),
        SD_BUS_METHOD("DisableMetrics", "", "", manager_method_metrics_disable, 0),
```
Both methods only return "Done" in a success case, which provides no additional information to the caller compared to just returning nothing - so this can be simplified.

Reply:
Ack